pkg/neorpc/types.go

```diff
@@ -17,10 +22,10 @@ const (
 )
 
 type (
-	// Raw represents JSON-RPC request. It's generic enough to be used in many
+	// Request represents JSON-RPC request. It's generic enough to be used in many
```

Comment:
Worth adding `request` to the commit message.

Reply:
Fixed.